index.ts

```diff
@@ -173,6 +173,7 @@ const moreThanHalfN = (n: bigint): boolean => n > (N >> 1n) // if a number is bi
 function getPublicKey(privKey: PrivKey, isCompressed = true) {   // Make public key from priv
   return Point.fromPrivateKey(privKey).toRawBytes(isCompressed);        // 33b or 65b output
 }
+type SignatureWithRecovery = SignatureWithRecovery
```

Comment:
huh?

Reply:
Oops, sorry.  Must have gotten caught up in a find and replace when I was cleaning up the PR after fiddling to get it working.
```suggestion
type SignatureWithRecovery = Signature & { recovery: number }
```